eds/plugins/vcs_provider_github.py

```diff
@@ -0,0 +1,141 @@
+import os
+from typing import Dict, Optional
+
+from github3 import login, enterprise_login
+from github3.github import GitHub, GitHubEnterprise
+from github3.orgs import Organization
+from github3.repos.repo import Repository
+from github3.repos.contents import Contents
+# from github3.repos.branch import Branch
+# from github3.git import CommitTree, Commit, Tree, Hash, Reference
+# from github3.repos.comparison import Comparison
+# from github3.users import User
+# from github3.exceptions import NotFoundError, ForbiddenError
+
+from eds.plugin import BasePlugin
+from eds.interfaces.vcs_provider import VcsProvider
+
+
+class GithubProviderPlugin(BasePlugin, VcsProvider):
```

Comment:
@qwcode Is this how I should be doing the inheritance? (`GithubProviderPlugin` is both a `BasePlugin` and implements the `VcsProvider` interface)

Reply:
yes, that is what I am doing.